tests/integration/project/elements/sandbox/remote-apis-socket.bst

```diff
@@ -0,0 +1,13 @@
+kind: manual
+
+depends:
+  - filename: base.bst
+    type: build
+
+sandbox:
+  remote-apis-socket:
+    path: /tmp/reapi.sock
+
+config:
+  build-commands:
+    - test -e /tmp/reapi.sock
```

Comment:
Can't we test that it's a socket instead?

```suggestion
    - test -S /tmp/reapi.sock
```

Reply:
Good suggestion, changed.